cmd/kubelet/app/server.go

```diff
@@ -206,6 +207,10 @@ HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 				klog.Fatal(err)
 			}
 
+			if (kubeletConfig.KubeletCgroups == "" && kubeletConfig.KubeReservedCgroup != "") || (0 != strings.Index(kubeletConfig.KubeletCgroups, kubeletConfig.KubeReservedCgroup)) {
```

Comment:
Using `strings.Contains(` would be easier to read here.

Reply:
I don't think we should omit an error when `KubeletCgroup` is not set, but `KubeReservedCgroup` is set.  Since the default for KubeReservedCgroup is `""`, they may just be setting `KubeReservedCgroup` to be the same as `KubeReservedCgroup`.

Reply:
thanks for your reminder，i open a new pr #72671 and close this one. in new one, KubeletCgroup will be set the same as KubeReservedCgroup as you said. but i don't use strings.Contains. for example, KubeletCgroup="/C/B/A/kubelet.service, KubeReservedCgroup="/B/A", i think that “strings.Contains” can not handle this situation，so i kept "string.index".